src/scenarios/weblarge2.0/pre.py

```diff
@@ -9,5 +9,5 @@
 
 setup_loggers(True)
 precommands = PreCommands()
-precommands.existing(os.path.join(sys.path[0], const.SRCDIR), 'mvc\\mvc.csproj')
+precommands.existing(os.path.join(sys.path[0], const.SRCDIR), 'mvc/mvc.csproj')
```

Comment:
Is a better fix to move this path component (`mvc` and `mvc.csproj`) inside the `os.path.join` call?